code/entry.py

```diff
@@ -1,79 +1,98 @@
         print("Successfully connect to database!")
         connectStatus = True
         conn.autocommit = True
     except:
         print("Fail to connect to the database, please try again!")
 
-if (connectStatus):
+if connectStatus:
     cur = conn.cursor()
-    while(True):
-        print("Select your sql file (sql contains no comment!): ")
-        query_path = easygui.fileopenbox()
-        file_name = query_path.split("\\")[-1].split(".")[0]
+    while True:
+        print("\n*** Select your sql file (sql contains no comment!): ")
+        query_path = easygui.fileopenbox(default=os.path.join('..', 'data', 'sql', "*"))
+        file_name = query_path.split(os.path.sep)[-1].split(".")[0]
+        print("Your sql file path: ", query_path)
         query = open(query_path, "r", encoding="utf8").read()
-        print("\nShow me the databases:\n")
 
+        # print("\nShow me the databases:\n")
         # command = query.split(' ')[0]
-        # if (command in {'SELECT', 'INSERT', 'UPDATE', 'DELETE', 'VALUES', 'EXECUTE', 'DECLARE'}):
+        # if command in ['SELECT', 'INSERT', 'UPDATE', 'DELETE', 'VALUES', 'EXECUTE', 'DECLARE']:
         #     query_qep = 'EXPLAIN (FORMAT JSON) ' + query
-        # elif (command == 'CREATE' and query.split(' ')[1] == 'TABLE'):
+        # elif command == 'CREATE' and query.split(' ')[1] == 'TABLE':
         #     query_qep = 'EXPLAIN (FORMAT JSON) ' + query
         # print(query_qep)
 
         query_array = query.split(';')
         qep_index = 0
         for query in query_array:
-            if query and (not query.isspace()):
-                # print(query)
-                query = query.strip()
-                query_qep = ''
-                command = query.split(' ')[0]
-                if command in {'SELECT', 'INSERT', 'UPDATE', 'DELETE', 'VALUES', 'EXECUTE', 'DECLARE'}:
-                    query_qep = 'EXPLAIN (FORMAT JSON) ' + query
-                elif command == 'CREATE' and query.split(' ')[1] == 'TABLE':
-                    query_qep = 'EXPLAIN (FORMAT JSON) ' + query
-                if len(query_qep) > 0:
-                    cur.execute(query_qep)
-                    rows = cur.fetchall()
-                    for row in rows:
-                        qep_index += 1
-                        with open('../data/json/' + file_name + '_' + str(qep_index) + '.json', 'wt') as out:
-                            res = json.dump(row[0], out, sort_keys=True, indent=4, separators=(',', ': '))
+            if not query or query.isspace():
+                continue
+            print("Query:", query)
+            query = query.strip().lower()
+            query_qep = ''
+            command = query.split(' ')[0]
+            if command in ['select', 'insert', 'update', 'delete', 'values', 'execute', 'declare']:
+                query_qep = 'explain (format json) ' + query
+            elif command == 'create' and query.split(' ')[1] == 'table' and query.split(' ')[2] == "as":
```

Comment:
and query.split(' ')[2] == "as"
Logic ko đúng vì syntax của code là 'CREATE TABLE MY_TABLE AS', nên query.split('')[2] sẽ là 1 table name chứ ko phải "AS". A cứ bỏ chỗ này đi thôi

Reply:
vẫn cần cái as tại create table cũng ko explain đc, sửa index